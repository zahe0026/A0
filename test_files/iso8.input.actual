test_files/iso8.input: ISO-8859-1 text
