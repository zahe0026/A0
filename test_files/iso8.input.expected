test_files/iso8.input: ISO-8859 text
